src/Window.zig

```diff
@@ -3,14 +3,14 @@ const ziglyph = @import("ziglyph");
 const WordIterator = ziglyph.WordIterator;
 const GraphemeIterator = ziglyph.GraphemeIterator;
 
-const Screen = @import("Screen.zig");
+pub const Screen = @import("Screen.zig");
```

Comment:
Let's drop this `pub`. I previously exported `Screen` via `main.zig` so this isn't needed